Assignment-1-6/form.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+include __DIR__ . '/email_validate.php';
+
+// Default value.
+$valid_phone = TRUE;
+$email_check = '';
+
+// If Form submitted.
+if (isset($_POST['submit'])) {
+  // Get first_name & last_name.
+  $first_name = $_POST['first-name'];
+  $last_name = $_POST['last-name'];
+
+  // Get image.
+  if (isset($_FILES['image']) && $_FILES['image']['error'] == UPLOAD_ERR_OK) {
+    $target_file = './images/' . basename($_FILES['image']['name']);
+
+    // Move uploaded image to 'images' folder.
+    if (move_uploaded_file($_FILES['image']['tmp_name'], $target_file)) {
+      $img_upload = 'successful';
+    }
+    else {
+      $img_upload = 'unsuccessful';
+    }
+  }
+
+  // Get sujects and marks string.
+  $marks_str = $_POST['subject-marks'];
+  // Convert the input string (sub|mark) in associative array of `sub => mark`.
+  $marks_arr = explode("\n", $marks_str);
+  $sub_mark_arr = array();
+
+  foreach ($marks_arr as $key => $mark) {
+    $sub_mark = explode('|', $mark);
```

Comment:
Our requirement was that user have to enter data only in the this format ;
Subject | Marks
We need to validate that , currently we are allowing any kind of data to be submitted, for example if user enters 40 | English, or maybe some special characters like ``$%# | $#! ` we are still printing it which isn't right. Please update accordinly with proper validations